app/src/main/res/values-es/strings.xml

```diff
@@ -98,8 +103,10 @@
     <string name="apply_success_dialog">Aplicación correctamente.\nPuede ser necesario reiniciar Android para que los cambios surtan efecto.\n\n¿Quieres reiniciar?\n(Para evitar la pérdida de datos, asegúrese de que ninguna aplicación use la memoria SD en este momento.)</string>
     <string name="apply_symlink_successful">Symlink fue creado correctamente.\nPuede ser necesario reiniciar Android para que los cambios surtan efecto.\n\n¿Quieres reiniciar?\n(Para evitar la pérdida de datos, asegúrese de que ninguna aplicación utiliza la memoria SD en este momento!)</string>
     <string name="apply_symlink_successful_title">El enlace simbólico se creó con éxito</string>
+    <string name="apply_symlink_fail">Symlink no pudo ser creado por Android.\nEsto falla principalmente debido a una \'característica\' llamada S-ON en los teléfonos HTC!\n\nUna solución es arrancar el teléfono en modo de recuperación y crear el symlink allí en -s /data/data/hosts /system/etc/hosts\'.\n\nSi esto no funciona buscar en Internet para S-OFF y su teléfono.</string>
     <string name="apply_symlink_fail_title">No se pudo crear el enlace simbólico</string>
     <string name="apply_help">Por favor lea la Ayuda para más información.</string>
+    <string name="apply_apn_proxy">¡Un proxy APN está configurado en su Android!\nAdAway no funcionará de manera confiable cuando se encuentre en redes móviles como 3G. Puede desactivar ese proxy al ir a su APN seleccionado (en la aplicación Configuración: Red e Internet - & gt; Red móvil - & gt; Avanzado - & gt; Nombres de puntos de acceso) y borre el valor en el campo proxy.</string>
```

Comment:
Make the build failed: `/home/travis/build/AdAway/AdAway/app/src/main/res/values-es/strings.xml:109:276: Resource and asset merger: The entity name must immediately follow the '&' in the entity reference.
    org.xml.sax.SAXParseException; lineNumber: 109; columnNumber: 276; The entity name must immediately follow the '&' in the entity reference.`

Reply:
I get these sometimes. Basically translators can really mess up any escapings or formattings.